guidance/guidance_los/scripts/los_guidance_action_server.py

```diff
@@ -324,9 +286,9 @@ def publish_reference(self, target):
         msg = PoseStamped()
         msg.header.frame_id = "odom"
         msg.header.stamp = self.get_clock().now().to_msg()
-        msg.pose.position.x = target[0]
-        msg.pose.position.y = target[1]
-        msg.pose.position.z = target[2]
+        msg.pose.position.x = target[0]  # x: x-coordinate of the target point
+        msg.pose.position.y = target[1]  # y: y-coordinate of the target point
+        msg.pose.position.z = target[2]  # z: z-coordinate of the target point
```

Comment:
If you utilize [Point](https://docs.ros.org/en/api/geometry_msgs/html/msg/Point.html) for target (thus, generally for waypoints), this could be simplified to

```python
def publish_reference(self, target: Point):
    ...
    msg.pose.position = target